.travis.yml

```diff
@@ -67,6 +67,10 @@ matrix:
           env: TOXENV=py27 CRYPTOGRAPHY_OSX_NO_LINK_FLAGS=0
         - language: generic
           os: osx
+          osx_image: xcode8
+          env: TOXENV=py35-cffi-min CRYPTOGRAPHY_OSX_NO_LINK_FLAGS=1
```

Comment:
Yes, but now that I look at the code that's because we have conditional branches for `update_into` on both the commoncrypto and openssl backends. We could just assume we have a sufficiently new version in those classes but then do the version check at the `CipherContext` layer in hazmat.primitives.ciphers.base. That would simplify this whole thing.